bangun_ruang.php

```diff
@@ -302,6 +302,17 @@ function volumeSetengahBola($jariJari)
         $EMPATPERTIGA = 4 / 3;
         return $SATUPERDUA * $EMPATPERTIGA * $this->PHI * $jariJari * $jariJari * $jariJari;
     }
+
+    /**
+     * Menghitung luas selimut kerucut
+     * rumus : phi * jari-jari * sisi
+     */
+
+    function LuasSelimutKerucut($PHI, $jariJari, $sisi)
+    {
+        $DUA = 2;
+       return $DUA * $this->$PHI * $jariJari * $sisi;
```

Comment:
coba perhatikan antara $this->PHI dengan $PHI di parameter. ada yg ganjil kan? coba lihat contoh2 fungsi yg lain  pasti gak seperti itu...